db/migrations/public/20240430133702_water-add-licence-agreements-view.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const viewName = 'licence_agreements'
+
+exports.up = function (knex) {
+  return knex
+    .schema
+    .createView(viewName, (view) => {
+      view.as(knex('licence_agreements').withSchema('water').select([
+        'licence_agreement_id AS id',
+        'financial_agreement_type_id AS financial_agreement_id',
+        'licence_ref',
+        'start_date',
+        'end_date',
+        'date_signed',
+        'date_deleted',
```

Comment:
Sorry, @robertparkinson 😳 

Missed this on first pass. We alias this to `deleted_at` to be consistent with `created_at` and `updated_at`.